make: /bin/bash: No such file or directory

### Problem description

Steps to reproduce behavior:
1. `bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/master/utils/installer/install.sh)`
2. Ignore dependency installation due to managing them through local Nix module.
3. Error arises from wrong Makefile `SHELL` entry. 
    a. Assuming it should be `/usr/bin/env bash` rather than `/bin/bash`.

### LunarVim version

Master (?)

### Neovim version (>= 0.6.1)

NVIM v0.6.1

### Operating system/version

NixOS 22.05 (Quokka)

### Relevant log output

```shell
Installing LunarVim shim
make: Entering directory '/home/icy-thought/.local/share/lunarvim/lvim'
make: /bin/bash: No such file or directory
make: *** [Makefile:8: install-bin] Error 127
make: Leaving directory '/home/icy-thought/.local/share/lunarvim/lvim'
```


### Screenshots

_No response_

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-SHELL := /bin/bash
+SHELL := /usr/bin/env bash
 
 install:
 	@echo starting LunarVim installer

